lldb/test/Shell/ObjectFile/ELF/PT_TLS-overlap-PT_LOAD.yaml

```diff
@@ -1,8 +1,25 @@
 # Overlapping PT_LOAD and PT_TLS segments should be able to exist side by side.
```

Comment:
Maybe clarify this by adding "in an object file." on the end. So it's clear that lldb is not making use of them, just able to ignore them as the comment below explains.

As it is it could be taken as "exist side by side in lldb", which is the opposite of what we're doing here.